pkg/agent/openflow/client_windows.go

```diff
@@ -22,9 +22,13 @@ import (
 	"net"
 
 	"antrea.io/antrea/pkg/agent/openflow/cookie"
+	"antrea.io/antrea/pkg/agent/route"
 	binding "antrea.io/antrea/pkg/ovs/openflow"
 )
 
+// globalVirtualSVCMAC is used for gateway IP in ARP responder table about Windows ClusterIP Service with antrea-proxy.
```

Comment:
antrea-proxy -> AntreaProxy

Reply:
I think you should say it is "for gateway virtual IP", "to implement ClusterIP Service with AntreaProxy on Windows".

Reply:
As the MAC (and the virtual IP as well) is used only on the local Node, I feel we should remove "global" from the name.

Reply:
Updated.